File: AICodingWeb/Models/PaymentMethod.cs
using System;
using System.Collections.Generic;

namespace AICodingWeb.Models
{
    public class PaymentMethod
    {
        public int PaymentMethodID { get; set; }
        public string PaymentMethodName { get; set; }
        public int LastEditedBy { get; set; }
    }
} 

File: AICodingWeb/aicoding_web/app/models/payment_method.py
from datetime import datetime
from app import db
from app.models.base import BaseModel

class PaymentMethod(db.Model, BaseModel):
    """PaymentMethod model"""
    
    __tablename__ = 'payment_methods'
    
    PaymentMethodID = db.Column(db.Integer, primary_key=True)
    PaymentMethodName = db.Column(db.String(50), nullable=False)
    LastEditedBy = db.Column(db.Integer, db.ForeignKey('people.PersonID'), nullable=False)
    ValidFrom = db.Column(db.DateTime, nullable=False)
    ValidTo = db.Column(db.DateTime, nullable=False)
    
    def __repr__(self):
        return f'<PaymentMethod {self.PaymentMethodID}>' 